README.md

```diff
@@ -23,12 +23,9 @@ The algorithms in RAFT span the following general categories:
 
 RAFT provides a header-only C++ library and pre-compiled shared libraries that can 1) speed up compile times and 2) enable the APIs to be used without CUDA-enabled compilers.
 
-RAFT also provides a Python library that is currently limited to
-1. a python wrapper around the `raft::handle_t` for managing cuda library resources
-2. definitions for using `raft::handle_t` directly in cython
-3. tools for building multi-node multi-GPU algorithms that leverage [Dask](https://dask.org/)
-
-The Python API is being improved to wrap the algorithms and primitives from the categories above.
+RAFT also provides 2 Python libraries:
+- `pylibraft` - cython wrappers around RAFT algorithms and primitives.
+- `pyraft` - reusable infrastructure for building analytics, such as tools for building multi-node multi-GPU algorithms that leverage [Dask](https://dask.org/).
```

Comment:
Might be good to specify exactly what is contained in each of these, it's a change even for existing functionality that seems to have moved like interruptible execution so it'd be very helpful